decoder/impeghd_hoa_space_positions.c

```diff
@@ -183,7 +183,10 @@ impeghd_hoa_ren_space_positions_init_with_param(ia_render_hoa_space_positions_st
       {
         continue;
       }
-
+      else if (ref_spk_layout->cicp_spk_idx[i] == 8)
```

Comment:
Why is it only for index 8, error code is added?  All other are supported?

Reply:
the geometric position for cicp index value 8 is not applicable. 
reference : ISO/IEC 23091-3, table 3.